src/validations/rules/sap.sch

```diff
@@ -479,6 +487,20 @@
                 statements.</sch:assert>
 
         </sch:rule>
+        
+        <sch:rule
+            context="oscal:terms-and-conditions/oscal:part[@name eq 'excluded-activities']">
+            
+            <sch:assert
+                diagnostics="has-roe-excluded-activity-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.19"
+                fedramp:specific="true"
+                id="has-roe-excluded-activity"
+                role="error"
+                test="oscal:part[@name eq 'excluded-activity']">The SAP Rules of Engagement (ROE) excluded activities have one or more excluded
```

Comment:
```suggestion
                test="oscal:part[@name eq 'excluded-activity']">The SAP Rules of Engagement (ROE) excluded activities must have one or more excluded
```

Reply:
I saw a couple of these in other issues I am working on and thought "darn, Dan is gonna find these and comment on them in the PR".

Fixed.